docker/start_server.sh

```diff
@@ -43,7 +55,12 @@ else
     echo "Server already initialized"
 fi
 
+echo "Wait for database to be ready"
+
+/wait
```

Comment:
Not completely sure, but it looks like this might be causing the following now:
> [Warning] Aborted connection 3 to db: 'unconnected' user: 'unauthenticated' host: '172.18.0.5' (This connection closed normally without authentication)

I assume that can be ignored, but in case this really comes from `wait`, maybe there is a way to have it properly close the connection?

Reply:
This doesn't matter at all. This is just how it works.